Program crashed in file: C:\GitLAbbur\SODA\Labb 01 - BattleERY\DaBattelry\CommonUtilities\CU/GridArray/GridArray.h
At line: 73
In function: CommonUtilities::GridArray<struct Tile,unsigned int>::Access
Trying to access GridArrayPosition that is out of range

---------------

Stack Crash Log Below:
---------------
c:\gitlabbur\soda\labb 01 - battleery\dabattelry\commonutilities\cu\dldebug\dl_debug.cpp (94): DL_Debug::Debug::AssertMessage

---------------
c:\gitlabbur\soda\labb 01 - battleery\dabattelry\dabattelry\gameobjects\unit\baseunitclass.cpp (26): BaseUnitClass::BaseUnitClass

---------------
c:\gitlabbur\soda\labb 01 - battleery\dabattelry\dabattelry\game\gameloop.cpp (130): GameLoop::InitUnits

---------------
c:\gitlabbur\soda\labb 01 - battleery\dabattelry\dabattelry\dabattelry.cpp (37): StartGame

---------------
c:\gitlabbur\soda\labb 01 - battleery\dabattelry\dabattelry\dabattelry.cpp (46): wmain

---------------
f:\dd\vctools\crt\crtw32\dllstuff\crtexe.c (623): __tmainCRTStartup

---------------
ERROR: SymGetLineFromAddr64, GetLastError: 487 (Address: 74AF919F)

---------------
74AF919F (KERNEL32): (filename not available): BaseThreadInitThunk

---------------
ERROR: SymGetLineFromAddr64, GetLastError: 487 (Address: 76EFA8CB)

---------------
76EFA8CB (ntdll): (filename not available): RtlInitializeExceptionChain

---------------
ERROR: SymGetLineFromAddr64, GetLastError: 487 (Address: 76EFA8A1)

---------------
76EFA8A1 (ntdll): (filename not available): RtlInitializeExceptionChain

---------------
